server/pages/index.jade

```diff
@@ -11,10 +11,8 @@ doctype html
 
 html(lang=lang, dir=isRTL ? 'rtl' : 'ltr', class=isFluidWidth ? 'is-fluid-width' : '')
 	head
-		if helmetTitle
```

Comment:
This condition is still necessary. If I navigate to e.g. the editor locally in this branch, there's an empty `<title></title>` in the `<head>`. Either that or we add the default to Redux and pull it from there in `server/render`.

![title](https://cloud.githubusercontent.com/assets/1779930/18992715/da030caa-86ed-11e6-99fc-5e06ae19bcb9.png)